### PR TITLE
Added the router chain test for cross chain communication

### PR DESCRIPTION
**Pull request by hydrogenbond007**
if you wanna check out more https://devnet-docs.routerprotocol.com/overview

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by alokeutpal — APPROVED** [review]
Y/l

**Comment by hydrogenbond007** [comment]
Just keeping it alive

**Comment by ligi** [comment]
https://aplha-explorer-ui.routerprotocol.com is not reachable

**Comment by hydrogenbond007** [comment]
> https://aplha-explorer-ui.routerprotocol.com is not reachable

Sorry it was a typo, fixed it

**Comment by hydrogenbond007** [comment]
@ligi ready to merge!

File: _data/chains/eip155-101010 [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "infoURL": "https://router-chain-docs.vercel.app/",
+  "shortName": "ROUTE",
+  "chainId": 9601,
+  "networkId": 9601,
+  "slip44": 701,
+  "explorers": [
+    {
+      "name": "Routerscan",
+      "url": "https://alpha-explorer-ui.routerprotocol.com",
+      "standard": "EIP3091"
```
Comment (ligi) [inline _data/chains/eip155-101010:25]: Does not seem to be EIP3091 compatible
```suggestion
      "standard": "none"
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**